ISM43362/ISM43362.h

```diff
@@ -68,9 +76,10 @@ class ISM43362
     *
     * @param ap the name of the AP
     * @param passPhrase the password of AP
+    * @param ap_sec the security level of network AP
     * @return true only if ISM43362 is connected successfully
     */
-    bool connect(const char *ap, const char *passPhrase);
+    bool connect(const char *ap, const char *passPhrase, uint32_t ap_sec);
```

Comment:
why not using ism_security_t as the type of the function ? 
this would avoid the cast and make sure that proper types are used on caller side.

Reply:
Done in a cleaner way.